config/locales/en/pages.yml

```diff
@@ -1,5 +1,31 @@
 en:
   pages:
+    "404":
+      title: This page cannot be found
+      body_html: |
+        <p>You can <a href="/">start a new prison visit request</a> or book
+        your visit by phone instead. See the
+        <a href="http://www.justice.gov.uk/contacts/prison-finder">prison
+        finder</a> for contact details.</p>
+    "500":
+      title: We are experiencing technical difficulties
+      body_html: |
+        <p>The prison visits booking system is not available right now, but
+        we're working hard to get things up and running again.</p>
+
+        <p>You can try again later or book your visit by phone instead. See the
+        <a href="http://www.justice.gov.uk/contacts/prison-finder">prison
+        finder</a> for contact details.</p>
+    "503":
+      title: We are experiencing technical difficulties
+      body_html: |
+        <p>The prison visits booking system is not available as we're carrying
+        out some planned maintenance. You can try again later or book your
+        visit by phone instead.</p>
+
+        <p>You can try again later or book your visit by phone instead. See the
+        <a href="http://www.justice.gov.uk/contacts/prison-finder">prison
+        finder</a> for contact details.</p>
```

Comment:
I'm thinking about translation, mainly, and I think there are several reasons to prefer to break it up:

1. It allows us to change structural or presentational (it exists, alas) HTML separately from translations.
2. Translators have less chance to break the layout by accident.
3. If we change part of the copy, it's clearer that there's only a small section whose translation should be changed.